pyanaconda/modules/storage/partitioning/automatic.py

```diff
@@ -46,6 +47,11 @@ def __init__(self):
         self.request_changed = Signal()
         self._request = PartitioningRequest()
 
+    @property
+    def method(self):
```

Comment:
Please rename the `method` it's too general. It could be encryption method, formatting method and so on.  Please rename it to `partitioning_method` or something similar.

Reply:
Fixed.